docs/troubleshooting.md

```diff
@@ -0,0 +1,320 @@
+
+  ----
+  
+
+#### Cause 2: PCB is in bootloader/DFU mode.
+
+#### Solution: Unplug keyboard, and if the keyboard has a switch on the back make sure it is on the 0 position. Plug back in once the switch is on the 0 position.
+
+-   0 = use keyboard
+    
+-   1 = boot/dfu mode (cannot use)
+    
+
+  
+!!! Note
+    All brutalist series PCBs, Instant60/65, AN-C v2, DB60 (bakeneko PCB), Satisfaction75, Chimera65, and most if not all CK designed PCBs will have this switch on them.
+
+  
+
+Occasionally, the switch on the back of the PCB gets switched on accident, or it is on 1 by default. Turn over the PCB and make sure it’s in the correct position, which in most cases will be 0. If it’s in the wrong position, you might need to unplug/plug back in your keyboard once you have switched positions.
+
+  
+
+![](images/troubleshooting/switch-0-1.jpg)
+
+*Above image shows an example of a PCB switch in the 0 position for use.*
+
+---  
+  
+
+#### Cause 3: Loose/improper connection on USB port, JST cable, or JST header.
+
+#### Solutions:
+
+-   USB port: Make sure the usb cable is properly seated, and try a different USB cable and USB port. If the cable wiggles, it may also cause seemingly random disconnections.
+    
+-   JST cable: Make sure there are no kinks/bends on the cable. Also make sure the JST cable male ends have no bent pins, or anything else that might interfere.
+    
+-   JST port: Make sure there are no bent pins inside of the female JST port.
+    
+-   JST connection: Make sure the JST cable is properly seated, and all the pins on the JST female ports are soldered in place.
+    
+
+!!! Note
+    When inserting the JST cable into the connector, support the JST female header by holding it while they plug in the JST cable. The ports can be delicate, and while they can sometimes be reattached, that is not an ideal situation to be in.
+    
+![](images/troubleshooting/jst-cable-and-header.jpg)
+
+*Left image: Male JST end with no issues. Each end of the JST cable will have one of these connectors. Color of wires may vary.
+Right image: JST header (female), where the JST cable plugs into. The daughterboard will have one of these, as well as the PCB.*
+
+  
+
+![](images/troubleshooting/05-db-installed.jpeg)
+
+*Above image shows a properly seated JST cable inserted into the female port of a daughterboard. Note how far the cable is inserted, as well as how it is evenly inserted.* 
+
+----
+ 
+### Issue: Specific key/keys are not registering.
+
+- #### Before attempting any of these steps, test the connection by bridging the pads with tweezers. If the PCB is hotswap, request they remove the switch before shorting.

```

Comment:
wording - remove 
"request they"

Reply:
would use the word "bridging" or "testing" vs "shorting"

Reply:
changed to bridging, removed the old CS wording.